cpu/esp8266/vendor/esp-idf/nvs_flash/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 HAW Hamburg
```

Comment:
Should be moved to `nvs_flash/src`.

Reply:
Moved in e38f6154042eae0f189239e4e10a847321edae65